app/helpers/tasks_helper.rb

```diff
@@ -1,2 +1,87 @@
+# このモジュールの内部では、Task情報をActiveRecord Relationとして渡された時は、
+# tasksをmap(&:attributes)によってハッシュ化してからデータを扱う
 module TasksHelper
+  def show_date(time)
+    time.strftime('%Y年%m月%d日')
```

Comment:
this method is not needed, use i18n.
https://railsguides.jp/i18n.html#%E6%97%A5%E4%BB%98%E3%83%BB%E6%99%82%E5%88%BB%E3%83%95%E3%82%A9%E3%83%BC%E3%83%9E%E3%83%83%E3%83%88%E3%82%92%E8%BF%BD%E5%8A%A0%E3%81%99%E3%82%8B